pages/task/read/views/index.jsx

```diff
@@ -70,23 +71,25 @@ const Task = ({ task }) => {
       <div className="govuk-grid-row">
         <div className="govuk-grid-column-one-third">
           <ol className="section-navigation">
-            <li className="active"><a href="#">Some</a></li>
-            <li><a href="#">Navigation</a></li>
-            <li><a href="#">Here</a></li>
+            <li className="active"><a href="#training">Training and exemptions</a></li>
+            <li><a href="#procedures">Procedure Categories</a></li>
+            <li><a href="#endorse">Endorse Application</a></li>
           </ol>
         </div>
 
         <div className="govuk-grid-column-two-thirds">
-          <Form formatters={formatters} />
+          <Pil />
+          <div className="govuk-grid-column-two-thirds" id="endorse">
+            <Form formatters={formatters} />
+          </div>
```

Comment:
The nesting of these grid elements seems a bit off. We're nesting a lot of `two-thirds` containers inside each other, which we almost certainly don't want.